--- tools/lint/cpplint.bzl
+++ tools/lint/cpplint.bzl
@@ -63,7 +63,7 @@ def _add_linter_rules(
     # root package.  Projects that want to use exactly the Drake defaults can
     # alias Drake's config file into their top-level BUILD.bazel file.)
     cpplint_data = list(data)
-    cpplint_cfgs = ["//:CPPLINT.cfg"]
+    cpplint_cfgs = [Label("//:CPPLINT.cfg")]
     for x in native.glob(
         ["CPPLINT.cfg", "test/CPPLINT.cfg"],
         allow_empty = True,
@@ -76,10 +76,10 @@ def _add_linter_rules(
     # Google cpplint.
     py_test_isolated(
         name = name + "_cpplint",
-        srcs = ["@cpplint_internal//:cpplint"],
+        srcs = [Label("@cpplint_internal//:cpplint")],
         data = cpplint_data + source_labels,
         args = _EXTENSIONS_ARGS + source_filenames,
-        main = "@cpplint_internal//:cpplint.py",
+        main = Label("@cpplint_internal//:cpplint.py"),
         size = size,
         tags = ["cpplint", "lint"],
     )
